flow/activities/flowable_core.go

```diff
@@ -83,17 +83,11 @@ func (a *FlowableActivity) applySchemaDeltas(
 ) error {
 	tableSchemaDeltasCount := len(schemaDeltas)
 	if tableSchemaDeltasCount > 0 {
-		modifiedSrcTables := make([]string, 0, tableSchemaDeltasCount)
-		for _, tableSchemaDelta := range schemaDeltas {
-			modifiedSrcTables = append(modifiedSrcTables, tableSchemaDelta.SrcTableName)
-		}
-
 		if err := a.SetupTableSchema(ctx, &protos.SetupTableSchemaBatchInput{
-			PeerName:         config.SourceName,
-			TableIdentifiers: modifiedSrcTables,
```

Comment:
@heavycrystal afaict `TableIdentifiers` wasn't used by SetupTableSchema, making this code look a bit silly. Makes me worry I'm missing something here